### PR TITLE
Pam 26 frontend employee can request asset

### PR DESCRIPTION
**Pull request by saugat977**

### CODE REVIEW

File: src/modules/assets/components/RequestAsset.vue [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+  methods: {
+    formData() {
+      return {
+        name: this.name.value,
+        category: this.category.value,
+        isNewProduct: this.isNewProduct.value,
+      };
+    },
+    validateField(field) {
+      console.log('entered func');
```
Comment (asimnp) [inline src/modules/assets/components/RequestAsset.vue]: remove console.log

File: src/modules/assets/components/RequestAsset.vue [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+      if (field === "name") {
+        if (this.name.value.length <  3){
+          this.name.error = "asset name should be more than three letters long";
+          console.log(this.name.value.length);
```
Comment (asimnp) [inline src/modules/assets/components/RequestAsset.vue]: remove console.log

File: src/modules/assets/components/RequestAsset.vue [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+      if (this.name.error ) {
+        return (this.submission.message =
+          "Some fields are not filled properly");
+      }
+      if(this.category.value === "select a category"){
+        return (this.submission.message =
+          "Please select a category");
+      }
+      this.submission.message = "Sorry, you cannot request an asset right now"
+      // try {
```
Comment (asimnp) [inline src/modules/assets/components/RequestAsset.vue:60]: If the commented code is not required, please remove it.

Comment (saugat977) [reply src/modules/assets/components/RequestAsset.vue:60]: The commented code is required when backend is ready for integration

Comment (asimnp) [reply src/modules/assets/components/RequestAsset.vue:60]: OK then.

File: src/modules/assets/components/RequestAsset.vue (hunk 1 above)
Comment (saugat977) [reply src/modules/assets/components/RequestAsset.vue]: updated

File: src/modules/assets/components/RequestAsset.vue (hunk 2 above)
Comment (saugat977) [reply src/modules/assets/components/RequestAsset.vue]: updated

**Review by asimnp — APPROVED** [review]

**Pull request merged**